fluent/ExtrinsicStatusRune.ts

```diff
@@ -14,13 +14,16 @@ export class ExtrinsicStatusRune<out U1, out U2, out C extends Chain = Chain>
     super(_prime)
   }
 
-  logStatus(...prefix: unknown[]): ExtrinsicStatusRune<U1, U2, C> {
-    return this.into(ValueRune).map((rune) =>
-      rune.into(ValueRune).map((value) => {
-        console.log(...prefix, value)
-        return value
-      })
-    ).into(ExtrinsicStatusRune, this.extrinsic)
+  logStatus<X>(...prefix: RunicArgs<X, unknown[]>): ExtrinsicStatusRune<U1, U2, C> {
+    return Rune
+      .tuple([this.into(ValueRune), Rune.tuple(prefix).lazy()])
+      .map(([rune, prefix]) =>
+        rune.into(ValueRune).map((value) => {
+          console.log(...prefix, value)
+          return value
+        })
```

Comment:
This should use `.log()`

Reply:
Done

Reply:
Actually, does the placement of the prefix resolution matter (inside one of the inner Runes vs. wrapping `this` and `Rune.tuple(prefixes)` in another `Rune.tuple`, and then mapping their results into the inner runes)?

Reply:
Kind of but not sufficiently for me to care at the moment.

Reply:
Really I just need a `.trackMap` or `.metaMap` or whatever on `MetaRune` and we can use that.